indy_node/test/write_permission/test_revocation_write_permission.py

```diff
@@ -58,4 +93,30 @@ def test_client_can_send_revoc_reg_entry(looper,
     rev_reg_entry[VALUE][REVOKED] = [1, 2, 3, 4, 5]
     del rev_reg_entry[VALUE][PREV_ACCUM]
     rev_entry_req = sdk_sign_request_from_dict(looper, sdk_wallet_client, rev_reg_entry)
-    sdk_send_and_check([json.dumps(rev_entry_req)], looper, txnPoolNodeSet, sdk_pool_handle)
\ No newline at end of file
+    sdk_send_and_check([json.dumps(rev_entry_req)], looper, txnPoolNodeSet, sdk_pool_handle)
+
+
+def test_rev_reg_def_only_trustee_steward_trust_anchor_can_create(looper,
```

Comment:
It looks like here we check that a common client can not send RevocRegDef, but we do not check that Trustee, Steward or TrustAnchor can.
I think this test can be parametrized to send RevocRegDef for every role expecting that only sending by Trustee, Steward or TrustAnchor will succeed.